src/export/export.go

```diff
@@ -32,6 +32,9 @@ func ToDir(state *core.BuildState, dir string, targets []core.BuildLabel) {
 		if pkg.Name == parse.InternalPackageName {
 			continue // This isn't a real package to be copied
 		}
+		if pkg.Subrepo != nil {
+			continue // Don't copy subrepo BUILD files... they don't exist
```

Comment:
I think the behaviour is correct here, but the comment is misleading - we don't want to copy anything from the subrepo, we just want to have the same subrepo configuration, right? But subrepos do actually have BUILD files (real, in the case of please plugins, and generated in the case of go_repo)

Reply:
Yeah, fair enough. They don't exist in our source tree though so we don't want to export them. Updated :)